src/core/common/trickle_timer.cpp

```diff
@@ -71,7 +71,7 @@ void TrickleTimer::Start(uint32_t aIntervalMin, uint32_t aIntervalMax, Mode aMod
     mMode = aMode;
 
     // Initialize I to [Imin, Imax]
-    if (Imin == Imax)
+    if (Imin == Imax || aMode == kModeNormal)
```

Comment:
A new proposal for the value of first I in the new patch. Please have a review. thanks.